superset/charts/api.py

```diff
@@ -914,3 +916,43 @@ def import_(self) -> Response:
         )
         command.run()
         return self.response(200, message="OK")
+
+    @expose("/user_slices/")
+    @expose("/user_slices/<int:user_id>/")
```

Comment:
could we just use `api/v1/chart/?q=(filters:!((col:owners,opr:rel_m_m,value:<USER_ID>)),order_direction:desc,page:0,page_size:25)`

Reply:
mm, would that return favorited slices too?